cwm-data/src/main/java/org/cast/cwm/data/User.java

```diff
@@ -67,7 +67,13 @@ public class User extends PersistedObject implements Serializable, DataUser, Com
 
 	@Column(unique=true)
 	protected String subjectId;
-	
+
+	/**
+	 * Identifier for LTI reference.
+	 */
+	@Column(unique = true)
+	protected String ltiId;
```

Comment:
This one is also not necessarily globally unique.

Reply:
See comment above, the import prefixes all ids from the LTI platform with the site's id.